ui/raidboss/timeline.js

```diff
@@ -57,12 +57,22 @@ class Timeline {
     return text;
   }
 
+  GetReplacedCommon(text) {
+    let locale = this.options.Language || 'en';
+    let keys = Object.keys(commonReplacement);
+    for (let j = 0; j < keys.length; ++j) {
+      let re = new RegExp(keys[j], 'gi');
+      text = text.replace(re, commonReplacement[keys[j]][locale]);
+    }
+    return text;
+  }
+
   GetReplacedText(text) {
-    return this.GetReplacedHelper(text, 'replaceText');
+    return this.GetReplacedCommon(this.GetReplacedHelper(text, 'replaceText'));
```

Comment:
Can GetReplacedCommon be called within GetReplacedHelper or GetReplacedCommon just be part of that function?

I think if you move this code after the `if (!this.replacements)` in GetReplacedCommon, then you won't have the test error.  The issue is that the test is just importing a few files, and `this.options` is undefined because no options is being passed to the timeline constructor.  It might be worth having the timeline constructor set options to {} if it's not defined to avoid this sort of thing in the future too.

Reply:
Oh, yes. I first thought I need just one time to call this function, logically. So, I thought I need this separately. (like if I put inside of that, this function will called twice.)
but I don't know how this code works, so I changed to minimize impact of these changes.
So, I left everything same, but added new function but I remembered I decided not to add these code inside that. It doesn't have to be but I remembered only that so I made some... mess.